engine/crates/tracing/src/otel/exporter.rs

```diff
@@ -1,16 +1,16 @@
-use std::{str::FromStr, time::Duration};
-
+#[cfg(feature = "otlp")]
```

Comment:
This intentional?  Might have been worth making it a module attr if so `#![cfg(feature = "otlp")]`

Reply:
typo! good catch

Reply:
for a moment there I thought I had a typo in the #[cfg]. Moved it to `otel.rs` instead and removed it from here.